website/docs/reference/developer-readme.md

```diff
@@ -30,6 +30,11 @@ description: SynapseML Development Setup
     - Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
     - Note: if you want to run conda commands from IntelliJ, you may need to select the option to add conda to PATH during installation.
     - Activate the `synapseml` conda environment by running `conda env create -f environment.yml` from the `synapseml` directory.
+    :::note
+    If you're using a Windows machine, please remove
+    `horovod==0.25.0` requirement in the environment.yml file, because horovod installation only
```

Comment:
If horovod version is upgraded in the future, this needs update again, better to just mention that remove `horovod` requirement from environment.yml